plugins/inputs/aos/aos_test.go

```diff
@@ -0,0 +1,48 @@
+package aos_test
+
+import (
+    "testing"
+    "os"
+
+    "github.com/influxdata/telegraf/plugins/inputs/aos"
+    "github.com/influxdata/telegraf/plugins/inputs/aos/aos_streaming"
+    "github.com/influxdata/telegraf/testutil"
+    "github.com/stretchr/testify/assert"
+)
+
+var aos_server string = os.Getenv("AOS_SERVER")
+
+func TestExtractAlertDataForProbeAlert(t *testing.T) {
```

Comment:
I'd suggest renaming this method to make it more specific and reflect the particular aspect it is validating. Ex: `TestExtractAlertDataForIntegerProbeAlert`.

Reply:
Yes - will continue improving test coverage.

Reply:
I meant to say rename existing test method name..